src/Elastic.Apm.AspNetCore/ApmMiddleware.cs

```diff
@@ -175,10 +192,10 @@ private void FillSampledTransactionContextRequest(HttpContext context, Transacti
 						Encrypted = context.Request.IsHttps, RemoteAddress = context.Connection?.RemoteIpAddress?.ToString()
 					},
 					HttpVersion = GetHttpVersion(context.Request.Protocol),
-					Headers = GetHeaders(context.Request.Headers, transaction.ConfigSnapshot),
+					Headers = GetHeaders(context.Request.Headers, transaction.ConfigSnapshot)
 				};
 
-				transaction.CollectRequestBody( /* isForError: */ false, context.Request, _logger);
+				await transaction.CollectRequestBody(false, context.Request, _logger);

```

Comment:
IMHO `/* isForError: */` is helpful - otherwise it's not clear what `false` argument means.

Reply:
I'm not sure if I manually turned this on, or it's the default, but most IDEs show this to you these days:

<img width="470" alt="image" src="https://user-images.githubusercontent.com/1091853/69239961-b738a280-0b9b-11ea-8ed0-965c959a2122.png">